protos/action/action.proto

```diff
@@ -177,6 +183,18 @@ message GotoLocationResponse {
     ActionResult action_result = 1;
 }
 
+message DoOrbitRequest {
+    float radius_m = 1; // Radius of circle (in meters)
+    float velocity_ms = 2; // Tangential velocity (in m/s)
+    float yaw_behavior = 3; // Yaw behavior of vehicle (ORBIT_YAW_BEHAVIOUR)
```

Comment:
Hmm this should not be a float, but an enum, corresponding to what PX4 implements from the [MAVLink definition](http://mavlink.io/en/messages/common.html#ORBIT_YAW_BEHAVIOUR).

Reply:
I updated the file to what I believe is the correct way to implement the enum.